ansible/roles/anitya-dev/tasks/main.yml

```diff
@@ -19,6 +19,7 @@
     - python3-devel
     - redhat-rpm-config
     - rpm-python
+    - python2-sqlalchemy_schemadisplay
```

Comment:
Is it possible to use the ```python3``` package instead, or is Anitya not yet Python 3 ready? Python 2 is deprecated in Fedora 30 and some packages are getting dropped there (not sure about schemadisplay).

Reply:
I will look if there is `python3` package for this library.

Reply:
Unfortunately there isn't `python3` package in Fedora 28.
However I see that the package maitainer is @jeremycline - Do you plan python3 package?

Reply:
I can look into it. Not sure why I didn't do a Python 3 package to begin with... Maybe it didn't support Python 3? I guess I'll find out